### ISSUE TITLE
Multiple input sources

### ISSUE THREAD

**Issue by reillyeon** [w3c/spec-prod#31]
Some specifications, such as [WebUSB](https://github.com/wicg/webusb) and [Web Bluetooth](https://github.com/webbluetoothcg/web-bluetooth) have multiple Bikeshed documents.

**Comment by xfq** [comment]
Other examples:

* https://github.com/w3c/csswg-drafts (Bikeshed)
* https://github.com/w3c/epub-specs/ (ReSpec)
* https://github.com/w3c/iip/ (ReSpec)

**Comment by sidvishnoi** [comment]
Supporting build/validate/deploy of multiple sources within the action will require major work.

I think we can add a config option `DESTINATION`, corresponding to `SOURCE`. Then the action can be called multiple times with different SOURCE/DESTINATION pairs, like:
```yml
runs-on: ubuntu-20.04
steps:
  - uses: actions/checkout
  - uses: w3c/spec-prod@v2
    with:
      SOURCE: specs/spec1/index.bs
      DESTINATION: spec1/index.html
      # other-inputs...
  - uses: w3c/spec-prod@v2
    with:
      SOURCE: specs/spec2/index.bs
      DESTINATION: spec2/
      # other-inputs...
```

It's a bit cumbersome if there are many specs, but it's one-time config.

It might be possible to reduce the boilerplate (with an added advantage of parallelism):
```yml
runs-on: ubuntu-20.04
strategy:
  matrix:
    include:
      - source: specs/spec1/index.bs
        destination: spec1/
      - source: specs/spec2/spec.bs
        destination: spec2/
steps:
  - uses: actions/checkout
  - uses: w3c/spec-prod@v2
    with:
      SOURCE: ${{ matrix.source }}
      DESTINATION: ${{ matrix.destination }}
      # other-inputs...
```

Notes to self:
- Each call to `spec-prod` will need clean-up after itself. 
- Avoid race conditions when deploying to GitHub pages (maybe `pull --rebase` right before `push --force-with-lease` can help)

**Comment by reillyeon** [comment]
My only concern with that is that it will generate a lot of noise in the gh-pages branch since each file updated will be its own commit.

**Comment by sidvishnoi** [comment]
https://github.com/w3c/spec-prod/issues/8 might help with reducing the noise. There will be multiple commits only when multiple specs are modified in same commit/push.

**Comment by jakearchibald** [comment]
https://github.com/w3c/ServiceWorker also has multiple input documents

**Comment by reillyeon** [comment]
Please push an new release with this change applied. I just tried enabling this for https://github.com/WICG/webusb and discovered too late that v2.2.3 doesn't include this change and so each step overwrites the previously deployed document.

**Comment by sidvishnoi** [comment]
Released [v2.3.0](https://github.com/w3c/spec-prod/releases/tag/v2.3.0)!
